test/src/receiver/main_receiver.cpp

```diff
@@ -20,10 +20,14 @@
 #include <cadmium/engine/pdevs_dynamic_runner.hpp>
 #include <cadmium/logger/tuple_to_ostream.hpp>
 #include <cadmium/logger/common_loggers.hpp>
-#include "../../../lib/vendor/NDTime.hpp"
+#include "../../../lib/DESTimes/include/NDTime.hpp"
 #include "../../../lib/vendor/iestream.hpp"
 #include "../../../include/data_structures/message.hpp"
 #include "../../../include/atomics/receiver_cadmium.hpp"
+#include "../../../include/output_modification/convert_output.hpp"
+
+#define SIMULATOR_OUTPUT_PATH "./test/data/receiver_test_output.txt"
```

Comment:
name of #define is not self-explanatory. This is receiver component and #define statement must be consistent with it.

eg  RECEIVER_OUTPUT_FILE_PATH

Reply:
in line-77: you can use SIMULATOR_OUTPUT_PATH instead of a hardcoded file pathname.

static std::ofstream out_data("test/data/receiver_test_output.txt");

Note: I couldn't comment on line-77 directly as GitHub doesn't allow me to do so.

Reply:
changing code as per the comments and resolving